### PR TITLE
Added extract method for Global Reactions

### LINKED ISSUE #154: Etabs_Toolkit: Extract GlobalReactions

### PR DESCRIPTION
**Pull request by alelom**
### Issues addressed by this PR
Added extract method for Global Reactions.

Closes #154 

 ### Test files
https://burohappold.sharepoint.com/:u:/s/BHoM/ESIF-wBKmJNJhN_MPxLBVwgBnP6ZdLg9VgKXsOIENUMAhQ?e=2deKAf

### CODE REVIEW

File: Etabs_Adapter/Helpers/Results.cs [hunk 1]
```diff
@@ -509,6 +509,57 @@ private static List<string> CheckAndGetCases(cSapModel model, IList cases)
...
+            double gx = 0; double gy = 0; double gz = 0;
+
+            //Get out loadcases, get all for null list
+            loadcaseIds = CheckAndGetCases(model, cases);
+
+            model.Results.Setup.DeselectAllCasesAndCombosForOutput();
+
+            for (int loadcase = 0; loadcase < loadcaseIds.Count; loadcase++)
+            {
+                if (model.Results.Setup.SetCaseSelectedForOutput(loadcaseIds[loadcase]) != 0)
```
Comment (IsakNaslundBh) [inline Etabs_Adapter/Helpers/Results.cs]: Here it looks like you are calling the same thing twice?

would probably change this to something like:

```
                if (model.Results.Setup.SetCaseSelectedForOutput(loadcaseIds[loadcase]) != 0)
                {
                    Engine.Reflection.Compute.RecordWarning("Failed to setup result extraction for case " + loadcaseIds[loadcase]);
                }
```

Comment (alelom) [reply Etabs_Adapter/Helpers/Results.cs]: I see, however I think it should be more like in 1565868

Comment (IsakNaslundBh) [reply Etabs_Adapter/Helpers/Results.cs]: Ahh, missread the lines, looked the same, sorry about that. That all makes sense now! Will test again and approve and merge if it works!

Comment (alelom) [reply Etabs_Adapter/Helpers/Results.cs]: No problem, thanks for testing! :)

**Pull request merged**